dashboard/test/ui/features/bee.feature

```diff
@@ -3,7 +3,6 @@ Feature: Complete a bee level
 Scenario: Complete Bee Conditions 4-5 Level 3
   Given I am on "http://learn.code.org/s/course3/stage/7/puzzle/3?noautoplay=true"
   When I rotate to landscape
-  And I wait to see "#x-close"
```

Comment:
This goes away in many places because I've rolled a "wait until the dialog is visible" step into the "And I close the dialog" step.